packages/tools/viewer/src/viewerFactory.ts

```diff
@@ -1,15 +1,15 @@
 // eslint-disable-next-line import/no-internal-modules
 import type { AbstractEngine, AbstractEngineOptions, EngineOptions, IDisposable, Nullable, WebGPUEngineOptions } from "core/index";
-
 import type { ViewerOptions } from "./viewer";
+
 import { Viewer } from "./viewer";
 
 /**
  * Options for creating a Viewer instance that is bound to an HTML canvas.
  */
 export type CanvasViewerOptions = ViewerOptions &
     (({ engine?: undefined } & AbstractEngineOptions) | ({ engine: "WebGL" } & EngineOptions) | ({ engine: "WebGPU" } & WebGPUEngineOptions));
-const defaultCanvasViewerOptions: CanvasViewerOptions = {};
+const defaultCanvasViewerOptions: CanvasViewerOptions = { antialias: true, adaptToDeviceRatio: true };

```

Comment:
Should we set the max ? I wonder if there is a default.

Reply:
You mean limitDeviceRatio? If so, I don't think so. Since we are using SceneOptimizer I think we should start at max resolution/quality and only down res as needed. For smaller and/or carefully crafted models, we often won't need to down res. Even for complex models we may not need to between WebGPU snapshot mode and/or auto suspending rendering.